<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Qt_Widgets_App_RoutineAutomatorClass</class>
 <widget class="QMainWindow" name="Qt_Widgets_App_RoutineAutomatorClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>470</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Malgun Gothic</family>
    <pointsize>12</pointsize>
    <bold>true</bold>
   </font>
  </property>
  <property name="focusPolicy">
   <enum>Qt::FocusPolicy::ClickFocus</enum>
  </property>
  <property name="windowTitle">
   <string>RoutineAutomator</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* 메인 창 배경 */
QMainWindow {
	background-color: #2b2b2b;
}

/* 서브 창 배경 */
QDialog {
	background-color: #2b2b2b;
}

/* 모든 텍스트 기본 설정 */
QWidget {
	color: #dcdcdc;
	font-family: &quot;Malgun Gothic&quot;; /* 한글 가독성 좋은 폰트 */
}

/* 구역을 나누는 선이나 테두리 (TreeWidget 등) */
QTreeWidget, QGroupBox, QLineEdit {
	background-color: #333333;
	border: 1px solid #a5d6a7; /* 연두색 선 */
	border-radius: 4px;
	gridline-color: #444444;
}

/* 헤더 부분 */
QHeaderView::section {
	background-color: #3d3d3d;
	color: #a5d6a7;
	padding: 5px;
	border: 0.5px solid #2b2b2b;
}

/* 버튼 스타일 */
QPushButton {
	background-color: #3d3d3d;
	border: 1px solid #a5d6a7;
	border-radius: 5px;
	padding: 5px 15px;
	min-height: 25px;
}

QPushButton:hover {
	background-color: #4d4d4d;
	border: 2px solid #c5e1a5; /* 호버 시 약간 더 밝은 연두 */
}

QPushButton:pressed {
	background-color: #2b2b2b;
}

/* 체크박스 */
QCheckBox::indicator {
	width: 15px;
	height: 15px;
	border: 1px solid #a5d6a7;
	background: #2b2b2b;
}

QCheckBox::indicator:checked {
	background-color: #a5d6a7; /* 체크 시 연두색으로 채움 */
}

/* 상태 표시줄 */
QStatusBar {
	background-color: #2b2b2b;
	color: #888888; /* 상태바 텍스트는 더 차분하게 */
}

/* 트레이 메뉴 및 모든 팝업 메뉴 설정 */
QMenu {
    background-color: #ffffff;      /* 메뉴 배경: 하얀색 */
    color: #000000;                 /* 텍스트 색상: 검은색 (사용자 요청) */
    border: 1px solid #dcdcdc;      /* 가느다란 회색 테두리 */
}

/* 메뉴 항목이 선택(마우스 오버) 되었을 때 */
QMenu::item:selected {
    background-color: #a5d6a7;      /* 우리가 정한 연두색으로 하이라이트 */
    color: #000000;                 /* 선택되어도 글자는 검은색 유지 */
}

/* 메뉴 내부의 구분선 */
QMenu::separator {
    height: 1px;
    background: #e0e0e0;
    margin: 5px;
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pbtn_addProc">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>11</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Malgun Gothic</family>
      <pointsize>13</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="focusPolicy">
     <enum>Qt::FocusPolicy::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>+ 추가</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pbtn_delProc">
    <property name="geometry">
     <rect>
      <x>146</x>
      <y>10</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Malgun Gothic</family>
      <pointsize>13</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>- 삭제</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbtn_upProc">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Malgun Gothic</family>
      <pointsize>13</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>↑ 위로 이동</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbtn_downProc">
    <property name="geometry">
     <rect>
      <x>435</x>
      <y>10</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Malgun Gothic</family>
      <pointsize>13</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>↓아래로 이동</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbtn_runProc">
    <property name="geometry">
     <rect>
      <x>588</x>
      <y>10</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Malgun Gothic</family>
      <pointsize>13</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>▶ 실행</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cb_autoStart">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>271</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Malgun Gothic</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>윈도우 시작 시 자동 실행 활성화</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="cb_trayIcon">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>461</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Malgun Gothic</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>트레이 아이콘으로 실행 유지 (체크 해제 시 완료 후 종료)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>390</y>
      <width>48</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Malgun Gothic</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>상태 : </string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_status">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>390</y>
      <width>601</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Malgun Gothic</family>
      <pointsize>12</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QTreeWidget" name="tw_procList">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>700</width>
      <height>258</height>
     </rect>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOn</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarPolicy::ScrollBarAlwaysOff</enum>
    </property>
    <property name="columnCount">
     <number>6</number>
    </property>
    <attribute name="headerCascadingSectionResizes">
     <bool>true</bool>
    </attribute>
    <column>
     <property name="text">
      <string>순서</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>타입</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>이름 / 경로</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>URL</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>지연</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>중복방지</string>
     </property>
     <property name="textAlignment">
      <set>AlignCenter</set>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Qt_Widgets_App_RoutineAutomator.qrc"/>
 </resources>
 <connections/>
</ui>
